src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -142,24 +141,32 @@ define([
                 this.$menu.find('.action').first().focus();
             },
 
+
+            focusLastItem: function() {
+                this.$menu.find('.action.last').focus();
```

Comment:
What about `this.$menu.find('.action').last().focus()`?  I'm not sure if the first item has a `first` class and it seems a bit inconsistent to do it differently here.  Just wondering!